src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/vsts_cd_provider.py

```diff
@@ -0,0 +1,81 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=no-self-use,too-many-arguments,too-many-lines
```

Comment:
There are indeed over style skipping in the legacy code base. It is not good practice and we're in the process of removing them.